tests/integration/telemetry/stackdriver/stackdriver_filter_test.go

```diff
@@ -184,9 +122,17 @@ func setupConfig(cfg *istio.Config) {
 	if cfg == nil {
 		return
 	}
+	cfg.ControlPlaneValues = `
+meshConfig:
+  enableTracing: true
+`
 	// enable stackdriver filter
 	cfg.Values["telemetry.v2.stackdriver.enabled"] = "true"
 	cfg.Values["telemetry.v2.stackdriver.logging"] = "true"
+	cfg.Values["telemetry.v2.stackdriver.topology"] = "true"
+	cfg.Values["global.proxy.tracer"] = "stackdriver"
+	cfg.Values["pilot.traceSampling"] = "100"
+	cfg.Values["telemetry.v2.stackdriver.configOverride"] = `{"enable_mesh_edges_reporting": true,"meshEdgesReportingDuration":"5s"}`
```

Comment:
unrelated to this PR: do we mix camel case and snake case intentionally? I noticed its this way in the envoyfilter as well?

Reply:
i do not believe it is intentional, but i can't recall the implications. As you guessed, this was copy-pasted from the envoyfilter, so I didn't even really think about it.